cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -228,6 +228,19 @@ func (p *pod) ID() string {
 	return p.id
 }
 
+func (p *pod) getCloneAnnotations(ctx context.Context, s *specs.Spec) (isTemplate bool, templateID string, err error) {
```

Comment:
Can this be moved to the oci package? The only thing pod is used for is just to check the OS and if it's hypervisor isolated which can both be done with checks to oci.IsWCOW and oci.IsIsolated. Something akin to oci.GetSandboxTypeAndID

Reply:
It is not possible to use `IsIsolated` during a container creation request. But I have moved this function into the oci package and added a wrapper around it here.